Header HEADER
string NAME

bool LMC_S
bool RMC_S
bool FMC_S

string ERR_DEV
string ERR_MSG

float32 LM_GAIN
int32 LM_SPD
float32 LW_SPD

float32 RM_GAIN
int32 RM_SPD
float32 RW_SPD

int32 FM_SPD
float32 FW_SPD

int32 FM_AGL
float32 FW_AGL
